app/src/main/res/values/strings.xml

```diff
@@ -385,4 +385,6 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_parkingType_surface">Normal</string>
     <string name="quest_parkingType_underground">Underground garage</string>
     <string name="quest_parkingType_multiStorage">Car park</string>
+    <string name="quest_toiletAvailability_title">Does this place have a toilet?</string>
```

Comment:
No, please require a name. How would the user know what "this place" is? Your screenshot is a good example of this, as the mall is not even a building outline.